neutone_sdk/audio.py

```diff
@@ -33,6 +73,24 @@ def __post_init__(self):
     def is_mono(self) -> bool:
         return self.audio.size(0) == 1
 
+    def to_mp3_bytes(self) -> bytes:
+        buff = io.BytesIO()
+        with tempfile.NamedTemporaryFile(suffix=".mp3") as temp:
+            write_mp3(temp.name, self.audio.T.numpy(), self.sr)
+            with open(temp.name, "rb") as f:
+                buff.write(f.read())
+        buff.seek(0)
+        return buff.read()
+
+    @classmethod
+    def from_file(cls, path: str) -> Self:
+        return cls.from_bytes(open(path, "rb").read())
+
+    @classmethod
+    def from_bytes(cls, _bytes: bytes) -> Self:
```

Comment:
Nit: the underscore should be at the end of the variable name to indicate it's not the reserved python keyword, otherwise at the front it implies it's private / only for internal use.

Reply:
Fixed